src/Core/Plugin/PluginBlockManagerTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace drunomics\ServiceUtils\Core\Plugin;
+
+use Drupal\Core\Block\BlockManagerInterface;
+
+/**
+ * Allows setter injection and simple usage of the service.
+ */
+trait PluginBlockManagerTrait {
+
+  /**
+   * The plugin block manager.
+   *
+   * @var \Drupal\Core\Block\BlockManagerInterface
+   */
+  protected $blockManager;
+
+  /**
+   * Sets the block manager.
+   *
+   * @param \Drupal\Core\Block\BlockManagerInterface $blockManager
+   *   The block manager.
+   *
+   * @return $this
+   */
+  public function setPluginBlockManager(BlockManagerInterface $blockManager) {
+    $this->blockManager = $blockManager;
+    return $this;
+  }
+
+  /**
+   * Gets the plugin block manager.
+   *
+   * @return \Drupal\Core\Entity\BlockManagerInterface
+   *   The block manager.
+   */
+  public function getPluginBlockManager() {
```

Comment:
let's set getBlockManager and BlockManagerTrait - the patterns is always {{CLASS}}Trait